train_GNN.job

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#SBATCH --partition=gpu_titanrtx_shared_course
+#SBATCH --gres=gpu:1
+#SBATCH --job-name=train_GNN
+#SBATCH --ntasks=1
+#SBATCH --cpus-per-task=3
+#SBATCH --time=05:00:00
+#SBATCH --mem=32000M
+#SBATCH --output=output/slurm_output_%A.out
+
+module purge
+module load 2019
+module load Python/3.7.5-foss-2019b
+module load CUDA/10.1.243
+module load cuDNN/7.6.5.32-CUDA-10.1.243
+module load NCCL/2.5.6-CUDA-10.1.243
+module load Anaconda3/2018.12
+
+# Your job starts in the directory where you call sbatch
+cd /home/lcur0321/ATCS_group1
```

Comment:
This should be $HOME or it breaks on any other account

Reply:
how exactly?

Reply:
I cannot access `/home/lcur0321/ATCS_group1` from my account `lcur0330`.
```bash
lcur0330@login3:$ cd /home/lcur0321/ATCS_group1
-bash: cd: /home/lcur0321/ATCS_group1: Permission denied
```

So you should use `cd $HOME/ATCS_group1` instead because `$HOME` always maps to the current user's home directory
```bash
lcur0330@login3:~/matyas/ATCS_group1$ $HOME
-bash: /home/lcur0330: Is a directory
```

Reply:
Perfect, thanks. Changed it